tests/test_inputinvoicemessagecontent.py

```diff
@@ -130,9 +130,8 @@ def test_to_dict(self, input_invoice_message_content):
             input_invoice_message_content_dict["max_tip_amount"]
             == input_invoice_message_content.max_tip_amount
         )
-        assert (
-            input_invoice_message_content_dict["suggested_tip_amounts"]
-            == input_invoice_message_content.suggested_tip_amounts
+        assert input_invoice_message_content_dict["suggested_tip_amounts"] == list(
```

Comment:
```suggestion
        assert input_invoice_message_content_dict["suggested_tip_amounts"] == tuple(
```
the class variable should become a tuple for this then

Reply:
I thought I had commented on that before, but I can't find it right now (except the bullet point in the PR description) … anyway, If a sequence-type attribute contains items, `to_dict` still converts that to a list. Converting to a tuple would require to either first create a list and then convert that to tuple or find a way to rephares the logic within `to_dict` as `tuple(intricate_logic(stuff) for stuff in sequence)`. Not sure if that's worth it, since the dict itself is mutable anyway and `to_dict` is mainly used to pass data to TG …

Reply:
yes I agree with you, probably not worth the effort, let's leave as is